code/web/src/modules/user/api/actions.js

```diff
@@ -87,6 +87,16 @@ export function register(userDetails) {
   }
 }
 
+export function updateUserInfo(userDetails) {
```

Comment:
Is updateUserInfo the function name we're agreeing on? This will need to be updated in BE if so